content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,46 @@
+# DVC Experiments Overview
+
+DVC Experiments are captures automatically by DVC when you [run them]. Each
+experiment creates and tracks a project variation based on the changes in your
+<abbr>workspace</abbr>. Experiments preserve the latest commit in the current
+branch (Git `HEAD`) as their parent or _baseline_.
+
+<details>
+
+### ⚙️ How does DVC track experiments?
+
+Experiments are custom [Git references] (found in `.git/refs/exps`) with a
+single commit based on `HEAD` (not checked out by DVC). Note that these commits
+are not pushed to Git remotes by default (see `dvc exp push`).
+
+</details>
+
+Experiments will have an auto-generated ID like `exp-bfe64` by default. A custom
+name can be given instead (using the `--name`/`-n` option of `dvc exp run`).
+
+> ID or name can be used to reference experiments with `dvc exp` subcommands.
+
+[run them]: /doc/user-guide/experiment-management/running-experiments
+[git references]: https://git-scm.com/book/en/v2/Git-Internals-Git-References
+
+## Basic workflow
+
+`dvc exp` commands let you automatically track a variation to a committed
+project version (baseline). You can create independent groups of experiments
+this way, as well as review, compare, and restore them later. The basic workflow
+goes like this:
+
+- Modify <abbr>parameters</abbr> or other dependencies (input data, source code,
+  stage definitions, etc.) of committed stages.
+- [Run experiments] with `dvc exp run` (instead of `repro`) to execute the
+  pipeline. The results are reflected in your <abbr>workspace</abbr>, and
+  tracked automatically.
+- Use [metrics](/doc/command-reference/metrics) to identify the best
```

Comment:
`dvc exp show` is used to identify the best experiments. Metrics provide a limited functionality compared to `exp show`.

Reply:
`exp show` displays metrics.

Reply:
I changed the link to /doc/user-guide/experiment-management/comparing-experiments now. I think that should resolve your concern.

Reply:
Never mind, rolling that back since we specifically mention `exp show` in the next bullet...

Reply:
Going back to this @iesahin , I merged the 2 bullets so the concern should be resolved now, I believe.